src/server/game/Battlefield/Zones/BattlefieldWG.h

```diff
@@ -455,6 +487,7 @@ class BattlefieldWG : public Battlefield
     }
 protected:
     bool m_isRelicInteractible;
+    TugOfWarWG* m_TugOfWar;
```

Comment:
https://en.cppreference.com/w/cpp/memory/unique_ptr

Reply:
Changed to unique_ptr
PTAL